### ISSUE TITLE
Compatability Issue with LethalExpansionCore

### ISSUE THREAD

**Issue by jrossa81** [HolographicWings/LethalExpansion#123]
I am not 100% sure if its an issue with LE itself or LEC, but when I have both loaded, none of the configurations stay.  They all reset to vanilla.

I have tested a base level profile that only had LE and LEC and they do not seem to be working well together any longer.

**Comment by HolographicWings** [comment]
the goal of LEC is to disable all the LE settings
tho LEC is outdated for now

**Comment by jrossa81** [comment]
Before, when I had both active, I could use the settings in the LE config file and still get all of the settings to work just fine (ie, extend deadline from 3 days to 5 days).

Now, well, sometime before Tuesday night, if I have both active, LEC seems to override the settings back to vanilla settings, (ie, even with config set to 5 days, it stays 3 days).

This never happened prior to me playing on Tuesday night.  Now, this could be simply because LEC has not updated in 3 weeks while LE has in the last week, but I wanted to see if you had any insight.

If not, I can post it LEC's page.

**Comment by HolographicWings** [comment]
Yes, unfortunately i can't help with LEC, that would take me too much time, it's not my mod and LEC's dev did an excellent work but altered and rewrote a lot of things

**Comment by TheOGButters** [comment]
Can we play without LEC? Half of my mods can't update bc they depend on LE

**Comment by HolographicWings** [comment]
> Can we play without LEC? Half of my mods can't update bc they depend on LE

why can't you play with LE ?
Today LEC help with the compatibility between LE and LLL but is outdated and don't have the last LE's features
